bin/doctor

```diff
@@ -345,7 +345,7 @@ function check_config_files() {
           print_point 2 "SHARELATEX_HISTORY_BACKEND: fs"
         fi
 
-        if [[ -n ${GIT_BRIDGE_ENABLED:-} ]]; then
+        if [[ ${GIT_BRIDGE_ENABLED:-false} = true ]]; then
           if [[ -n ${GIT_BRIDGE_OAUTH2_CLIENT_SECRET:-} ]]; then
```

Comment:
```suggestion
        if [[ "${GIT_BRIDGE_ENABLED:-false}" == "true" ]]; then
          if [[ "${GIT_BRIDGE_OAUTH2_CLIENT_SECRET:-null}" != "null" ]]; then
```
Nit: WDYT about aligning the code style for conditions with the rest of the file?

Reply:
Good idea, but I'd do it the other way around. These quotes are not necessary. This can be done in another PR.